plotly/tests/test_core/test_graph_objs/test_figure.py

```diff
@@ -16,7 +16,7 @@ def test_instantiation(self):
             'frames': []
         }
 
-        Figure(native_figure)
+        Figure(**native_figure)
```

Comment:
🚨 - Breaking Change

Reply:
FYI: `Figure` can now take in another figure, or a graph representing a figure, as the first arg.

Reply:
OK sounds good, let's revert back to the previous test then (cc @Kully )